Create 2021-11-22-IT-curriculum-update (#1233)
carpentries/carpentries.org: _posts/2021/11/2021-11-18-IT-curriculum-update.md

---
layout: page
authors: ["Karen Word"]

> teaser: "The new iteration of the Instructor Training curriculum has been submitted for release"

Reviewer: ```suggestion
teaser: "The new version of the Instructor Training curriculum has been released"

Just trying to make this a little catchier 
```

title: "The Carpentries Instructor Training Curriculum has been Updated!"
date: 2021-11-18
time: "09:00:00"
tags: ["Instructor Trainers"]
---

> Throughout most of 2021, The Carpentries Core Team and Trainer community have been developing, discussing, and testing an updated version of our Instructor Training curriculum. It has now been merged into our repository on GitHub and submitted to Zenodo for official release!

Reviewer: ```suggestion
Throughout 2021, The Carpentries Core Team and Trainer community have been developing, discussing, and testing an updated version of our Instructor Training curriculum. It has now been merged into our repository on GitHub and submitted to Zenodo for official release!
```

### Changes in this update include:
* General changes to make the curriculum more useable and maintainable:
    - [ ] Reduced hyperlinks to external websites 
    - [ ] Eliminated use of handouts for activities

>     - [ ] Reduced duplication of policy articulated elsewhere; increased orientation to resources

Reviewer: I don't feel like this line is clear to people who haven't been part of our conversations.

Author: ```suggestion
    - [ ] Reduced content on Carpentries policies; increased orientation to main sources of policy information
```

    - [ ] Reduced use of jargon
    - [ ] Reduced use of coding in examples to accommodate variable background knowledge
    - [ ] Structural changes to improve timing and flow, allowing for more consistent breaks & alternate schedules
* Content removed or transferred to [optional exercises page](https://carpentries.github.io/instructor-training/additional_exercises/index.html) (more relevant to lesson development, which is now a separate curriculum)
    - [ ] Practice writing multiple choice questions
    - [ ] Practice writing learner profiles
    - [ ] Content on Bloom's Taxonomy
* Content changed or rearranged
    - [ ] Previous episode on "Mindset" reduced and combined with ["Motivation and Demotivation" episode](https://carpentries.github.io/instructor-training/08-motivation/index.html)
    - [ ] Previous content on Equity, Inclusion, and Accessibility in "Motivation and Demotivation" episode moved to [its own episode](https://carpentries.github.io/instructor-training/09-eia/index.html) and expanded
    - [ ] Previous references to formative assessment sprinkled across episodes are now highlighted as distinct uses

>     - [ ] Mental models are now [explained using concept mapping](https://carpentries.github.io/instructor-training/02-practice-learning/index.html) with addition of analogies to summarize a recognizable model

Reviewer: I find the phrase "analogies to summarize a recognizable model" a little unclear

Author: ```suggestion
    - [ ] Mental models are now [explained using concept mapping](https://carpentries.github.io/instructor-training/02-practice-learning/index.html) and analogies
```

## How Do You Update a Curriculum In Use?
For this update we used an older GitHub account under The Carpentries umbrella, called "data-lessons". Working in a [fork](https://github.com/data-lessons/instructor-training) allowed us to separate comments on the lesson-in-development from those in the existing lesson. Changes were submitted as pull requests to the data-lessons repository, reviewed and merged there; the primary branch of that repository was used in a [pull request to our main repository](https://github.com/carpentries/instructor-training/pull/1215), which was then subjected to broader community review. Meanwhile, the [fully rendered curriculum](https://data-lessons.github.io/instructor-training/) in the data-lessons repository could be field tested with minimal disruption.

> This workflow was not entirely satisfactory. One effect was to narrow the audience for early conversations, which occurred in the "quieter" data-lessons repository and was mostly restricted to Maintainers and a few Trainers who were paying close attention. Some found this process to be too secretive, preferring that all conversation remain in our main repository. In addition, the many ways to provide feedback (issues or comments on the PR in the main repository; issues or comments on PRs in data-lessons) proved confusing for both contributors and maintainers. 

Reviewer: secretive doesn't feel like the right word. Maybe isolated? Maybe separated from our normal discussions? Maybe inaccessible?

Author: ```suggestion
This workflow was not entirely satisfactory. One effect was to narrow the audience for early conversations, which occurred in the "quieter" data-lessons repository and was mostly restricted to Maintainers and a few Trainers who were paying close attention. Some found this process to be too isolated, preferring that all conversation remain in our main repository. In addition, the many ways to provide feedback (issues or comments on the PR in the main repository; issues or comments on PRs in data-lessons) proved confusing for both contributors and maintainers. 
```
